openid-4-verifiable-presentations-1_0.md

```diff
@@ -1492,7 +1492,139 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
+
+The following is a non-normative example of how the W3C Digital Credentials API can be used with an unsigned OpenID4VP request when advanced security features of OpenID4VP are not used:
+
+```js
+if ('DigitalCredential' in window) {
+  const credential = await navigator.identity.get({
+    digital: {
+      providers: [{
+        protocol: "urn:openid.net:oid4vp",
+        request:  {
+          response_type: "vp_token",
+          nonce: "n-0S6_WzA2Mj",
+          client_metadata: {...},
+          presentation_definition: {...}
+        }
+      }]
+    }
+  });
+} else {
+// fallback to other invocation mechanisms
+}
+```
+
+Out of the Authorization Request parameters defined in [@!RFC6749] and (#vp_token_request), the following are supported with this profile: 
+
+* `client_id`
+* `client_id_scheme`
+* `response_type`
+* `nonce`
+* `presentation_definition`
+* `client_metadata`
+* `request`
+* `response_mode`
+
+The `client_id` and `client_id_scheme` MUST be omitted in unsigned requests defined in (#unsigned_request). The Wallet determines the Client Identifier from the origin as asserted by the Browser. How the Wallet receives the origin from the Browser is up to the user agent and app platform and is out of scope of this profile.
+
+The value of `response_mode` parameter MUST be `w3c_dc_api` when the response is neither signed nor encrypted and `w3c_dc_api.jwt` when the response is signed and/or encrypted as defined in (#jarm).
+
+In addition to the above-mentioned parameters, this profile introduces a new parameter:
+
+* `expected_origins`: REQUIRED when signed requests defined in (#signed_request) are send over the W3C Digital Credentials API [@!w3c.digital_credentials_api]. An array of strings, each of the string representing an origin of the Verifier that is making the request. The Wallet can detect replay of the request by comparing values in this parameter to the origin asserted by the user agent.
+
+## Signed and Unsigned Requests
+
+Any OpenID4VP request compliant to this section of this specification can be sent over the W3C Digital Credentials API. Depending on the mechanism used to identify and authenticate the Verifier, the request can be signed or unsigned. This section defines signed and unsigned OpenID4VP requests over the W3C Digital Credentials API.
+
+### Unsigned Request {#unsigned_request}
+
+The Verifier MAY send all the OpenID4VP request data as JSON elements in the `request` API parameter and receives the result in the API's `result` parameter. In this case, the Wallet will use the Verifier origin as asserted by the Browser as the Verifer's Client Identifier.
+
+The unsigned Request Object MUST NOT contain `client_id` and `client_id_scheme` parameters.
+
+### Signed Request {#signed_request}
+
+The Verifier MAY send a signed request, for example, when identification and authentication of the Verifier is required.
```

Comment:
I would go for a SHOULD.

Reply:
Resolving as per discussion on today's WG call:

The browser folks are pretty keen to say that unsigned requests are generally fine, the may reason we have signed requests here is because of EU law having a requirement to identify the verifier using a certificate. So we should leave this as a 'MAY'.